test/e2e/density.go

```diff
@@ -361,18 +367,29 @@ var _ = framework.KubeDescribe("Density", func() {
 		{podsPerNode: 50, runLatencyTest: false},
 		{podsPerNode: 95, runLatencyTest: true},
 		{podsPerNode: 100, runLatencyTest: false},
+		{podsPerNode: 30, runLatencyTest: true, kind: extensions.Kind("Deployment")},
```

Comment:
I wouldn't introduce those as predefined. I think that we should just mix all types (but not in this PR).